puma/metadata.csv

```diff
@@ -0,0 +1 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
```

Comment:
Can we include the metric name, type, unit_name (if applicable), description, integration and short name of the metrics submitted?

```suggestion
metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
puma.workers,gauge,,unit,,Number of puma workers,0,puma,puma worker
```

^example of the `puma.workers` metric